actor-typed-tests/src/test/scala-3/docs/org/apache/pekko/typed/InteractionPatterns3Spec.scala

```diff
@@ -0,0 +1,660 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Is this code based on Lightbend code or is it your own? If it is your own, could you use a standard Apache header?

An example - https://github.com/apache/incubator-pekko/blob/main/project/AddMetaInfLicenseFiles.scala#L1-L16

Reply:
As the new file is a modified version of file [InteractionPatternsSpec.scala](https://github.com/apache/incubator-pekko/blob/main/actor-typed-tests/src/test/scala/docs/org/apache/pekko/typed/InteractionPatternsSpec.scala) in the Pekko repo, the header is a verbatim copy of the header in that file.

I assume the Lightbend copyright header is there for a reason, unless it was left there in error. In the former case it should be kept in the new file, otherwise it should be removed in both files. Please advise.

Reply:
That's fine. It is correct to keep the headers in this case then.